kgtk/cli/build_faiss.py

```diff
@@ -0,0 +1,111 @@
+"""
+Train and populate a faiss index that can compute nearest neighbors of given embeddings.
+"""
+
```

Comment:
rename the command and the file in `cli` to build-faiss-index`

Reply:
done